_includes/navbar.html

```diff
@@ -177,7 +177,9 @@
 
               <a class="smalllinks" href="https://www.apache.org/licenses/" target="_blank">License</a> <small><span class="glyphicon glyphicon-new-window"></span></small>
 
-              <a class="smalllinks" href="https://www.apache.org/security/" target="_blank">Security</a> <small><span class="glyphicon glyphicon-new-window"></span></small>
+              <a class="smalllinks" href="{{ baseurl_i18n }}/security.html">Flink Security</a>
```

Comment:
This kinda ruins the current layout of the sidebar.

![Screenshot_2021-01-06 Apache Flink Stateful Computations over Data Streams](https://user-images.githubusercontent.com/5725237/103763130-b110ba00-5019-11eb-8021-5522db21c5b4.png)

Maybe we could throw out some other item in the sidebar instead, like the plan visualizer.

Reply:
Is this better?

![image](https://user-images.githubusercontent.com/89049/103791998-699f2380-5043-11eb-9c25-fb8f926b626c.png)

Reply:
it is _better_, but...like the current layout for the apache-mandated stuff is nigh perfect :/

Reply:
I vaguely remember that the plan visualizer is kinda popular (people complained when we removed it) .. that's why I want to keep it.

How about this? 
![image](https://user-images.githubusercontent.com/89049/103865534-a0694e00-50c4-11eb-8eca-78c11288d155.png)

Reply:
I think that's the better alternative at this time. (although I'm worried that over time the apache stuff gets pushed down more and more)

Reply:
I agree that we need to be careful. Thanks for confirming.